src/utils/daos_hdlr.c

```diff
@@ -226,7 +856,7 @@ cont_create_hdlr(struct cmd_args_s *ap)
 		attr.da_chunk_size = ap->chunk_size;
 		rc = dfs_cont_create(ap->pool, ap->c_uuid, &attr, NULL, NULL);
```

Comment:
so we don't want to set a property for a dfs container with POSIX type here?
similarly we probably need something similar in cont_create_uns_hdlr()

Reply:
good catch. the uns path hasn't been covered.

Reply:
done